### PR TITLE
Discourage users from toggling Tiered Storage off then on

### PR DESCRIPTION
**Pull request by kbatuigas**
## Description

Resolves https://github.com/redpanda-data/documentation-private/issues/2648
Review deadline:

Add an admonition (Caution), currently in the Limitations section but this can be moved elsewhere.

Another [PR](https://github.com/redpanda-data/docs/pull/583) had previously been merged to advise against repeatedly toggling remote write on and off. This will make the warning more prominent.

## Page previews

[Tiered Storage](https://deploy-preview-679--redpanda-docs-preview.netlify.app/current/manage/tiered-storage/#limitations)

## Checks

- [ ] New feature
- [x] Content gap
- [x] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

File: modules/manage/partials/tiered-storage.adoc [hunk 1]
```diff
@@ -881,7 +881,12 @@ rpk topic alter-config <topic_name> --set redpanda.remote.read=true --set redpan
...
 
-NOTE: If you disable remote writes and reads on a topic, that is, set `redpanda.remote.read` and `redpanda.remote.write` to `false`, the data that has been offloaded to object storage is not deleted. When you re-enable remote writes and reads, Redpanda resumes uploading data to object storage starting from the last write point, and the part of the log uploaded to object storage becomes available for reads again.
+[NOTE]
+====
+If you disable remote writes and reads on a topic, that is, set `redpanda.remote.read` and `redpanda.remote.write` to `false`, the data that has been offloaded to object storage is not deleted. When you re-enable remote writes and reads, Redpanda resumes uploading data to object storage starting from the last write point, and the part of the log uploaded to object storage becomes available for reads again.
+
+Redpanda strongly recommends that you do not re-enable Tiered Storage after previously enabling and disabling it, because this can result in inconsistent data and data gaps in Tiered Storage for a topic.
```
Comment (daisukebe) [inline modules/manage/partials/tiered-storage.adoc]: > If you disable remote writes and reads on a topic, that is, set `redpanda.remote.read` and `redpanda.remote.write` to `false`, the data that has been offloaded to object storage is not deleted. When you re-enable remote writes and reads, Redpanda resumes uploading data to object storage starting from the last write point, and the part of the log uploaded to object storage becomes available for reads again.

This implies TS is enabled, disabled, and can be enabled once again. This is conflicting with the new warning we want to add. Probably the current NOTE should be completely deleted. @mattschumpert thoughts?

Comment (kbatuigas) [reply modules/manage/partials/tiered-storage.adoc]: Exactly -- my preference would be to remove the note entirely.

File: modules/manage/partials/tiered-storage.adoc [hunk 2]
```diff
@@ -32,6 +32,8 @@ rpk cluster license info
 - Migrating topics from one object storage provider to another is not supported.
 - Migrating topics from one bucket or container to another is not supported.
 
+CAUTION: Redpanda strongly recommends that you do not re-enable Tiered Storage after previously enabling and disabling it, because this can result in inconsistent data and data gaps in Tiered Storage for a topic.
```
Comment (asimms41) [inline modules/manage/partials/tiered-storage.adoc]:
```suggestion
CAUTION: Redpanda strongly recommends that you do not re-enable Tiered Storage after previously enabling and disabling it. Re-enabling Tiered Storage can result in inconsistent data and data gaps in Tiered Storage for a topic.
```
Just split this into a couple of sentences. Just a suggestion that could be applied wherever you use this statement.

File: modules/manage/partials/tiered-storage.adoc [hunk 3]
```diff
@@ -879,7 +881,12 @@ rpk topic alter-config <topic_name> --set redpanda.remote.read=true --set redpan
 
 Topic-level properties override cluster-level properties. For example, for new topics, if `cloud_storage_enable_remote_write` is set to `true`, you can set `redpanda.remote.write` to `false` to turn it off for a particular topic.
 
-NOTE: If you disable remote writes and reads on a topic, that is, set `redpanda.remote.read` and `redpanda.remote.write` to `false`, the data that has been offloaded to object storage is not deleted. When you re-enable remote writes and reads, Redpanda resumes uploading data to object storage starting from the last write point, and the part of the log uploaded to object storage becomes available for reads again.
+[NOTE]
+====
+If you disable remote writes and reads on a topic, that is, set `redpanda.remote.read` and `redpanda.remote.write` to `false`, the data that has been offloaded to object storage is not deleted. When you re-enable remote writes and reads, Redpanda resumes uploading data to object storage starting from the last write point, and the part of the log uploaded to object storage becomes available for reads again.
```
Comment (asimms41) [inline modules/manage/partials/tiered-storage.adoc]:
```suggestion
If you disable remote writes and reads on a topic by setting `redpanda.remote.read` and `redpanda.remote.write` to `false`, the data that has been offloaded to object storage is not deleted. When you re-enable remote writes and reads, Redpanda resumes uploading data to object storage starting from the last write point, and the part of the log uploaded to object storage becomes available for reads again.
```

**Review by asimms41 — APPROVED** [review]
Just a couple of suggestions.

File: modules/manage/partials/tiered-storage.adoc (hunk 1 above)
Comment (kbatuigas) [reply modules/manage/partials/tiered-storage.adoc]: @daisukebe Matt confirmed via Slack that we should remove this note.

**Pull request merged**